projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php

```diff
@@ -241,6 +241,28 @@ function_exists( 'wp_theme_has_theme_json' ) ? wp_theme_has_theme_json() : WP_Th
 			'isBlazeDashboardEnabled'     => Blaze::is_dashboard_enabled(),
 			/** This filter is documented in plugins/jetpack/modules/subscriptions/subscribe-module/class-jetpack-subscribe-module.php */
 			'isSubscriptionModalEnabled'  => apply_filters( 'jetpack_subscriptions_modal_enabled', false ),
+			'socialInitialState'          => self::get_publicize_initial_state(),
+		);
+	}
+
+	/**
+	 * Gets the initial state for the Publicize module.
+	 *
+	 * @return mixed|null
```

Comment:
Should we change this here?

```suggestion
	 * @return array|null
```

Reply:
Nice catch thanks!